CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+# Contributing
+
+Thank you for your interest in contributing to the .NET documentation!
+
+This repository contains the samples used in the .NET documentation. The main repository for .NET documentation is the [.NET Docs repository](https://github.com/dotnet/docs). See the [Contributing Guide](https://github.com/dotnet/docs/tree/master/Contributing.md) for information on contributing samples and topics to the .NET documentation.
```

Comment:
I think that this should mention something about opening issues in the .NET Docs repository,